include/ck/tensor_operation/gpu/element/binary_element_wise_operation.hpp

```diff
@@ -18,6 +18,51 @@ struct Add
     {
         dst = src1 + src2;
     }
+
+    __host__ __device__ constexpr void
+    operator()(half_t& dst, const half_t& src1, const half_t& src2) const
+    {
```

Comment:
better to using template, avoiding data auto transfer

Reply:
Addressed by [d478a38](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/230/commits/d478a3893418a305ecc14596f6c068c8bc625034)